src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx

```diff
@@ -16,13 +16,13 @@
 
 import React from 'react';
 
-const TracksManagerModal = () => {
+const NavigateModal = () => {
   return (
-    <section className="tracksManagerModal">
-      <p>Save multiple browser configurations</p>
+    <section className="NavigateModal">
```

Comment:
What is the purpose of this CSS class?

Reply:
I have removed the css class. Its funny though all the other modal has the same thing.

Reply:
We need to remove them too if we aren't using them. But let's do that in another PR :)